documentation/commands/git-rename-branch.md

```diff
@@ -0,0 +1,37 @@
+#### NAME
+
+git-rename-branch - rename a branch both locally and remotely
+
+
+#### SYNOPSIS
+
+```
+git rename-branch <branchname> <newbranchname> [-f]
+```
+
+
+#### DESCRIPTION
+
+On a non-feature branch, requires the use of the `-f` option
+Syncs the repository if there is a remote repository
+Creates a branch with the new name based on the old name
+If there is a remote repository
+* pushes the new branch
+* deletes the old branch from the remote repository
+Deletes the old branch locally
+Reconfigures git-town locally if renaming a non-feature branch
```

Comment:
Lets follow this format for this paragraph:
* longer general description of the command
* edge cases and other things worth knowing
* list of the steps executed

This means

```
Renames the given branch on both the local machine and the remote if one is configured. 
This command is intended for feature branches. Renaming non-feature branches has to be confirmed with the `-f` option.

[ list of steps here ]
```